services/web/server/tests/integration/01/test_exporter_requests_handlers.py

```diff
@@ -0,0 +1,236 @@
+    .resolve()
+    .parent
+)
+
+
+async def _new_project(
+    client: TestClient,
+    user: UserInfoDict,
+    product_name: str,
+    project_template_file: Path,
+) -> ProjectDict:
+    """returns a project for the given user"""
+    project_data = empty_project_data()
+
+    assert client.app
+    return await create_project(
+        client.app,
+        project_data,
+        user["id"],
+        product_name=product_name,
+        default_project_json=project_template_file,
+    )
+
+
+def _get_files_in_zip(zip_path: Path) -> set[str]:
+    return {x.filename for x in ZipFile(zip_path).infolist()}
+
+
+def _get_fake_template_projects() -> set[Path]:
+    template_dir = CURRENT_DIR / "data"
+    assert template_dir.exists()
+    return set(template_dir.rglob("fake-template-projects*"))
+
+
+@pytest.fixture(params=_get_fake_template_projects())
+def template_path(request: FixtureRequest) -> Path:
+    return request.param
+
+
+@pytest.fixture
+def product_name() -> str:
+    return "osparc"
+
+
+@pytest.fixture
+def client(
+    docker_swarm: None,
+    event_loop: asyncio.AbstractEventLoop,
+    aiohttp_client: Callable,
+    app_config: dict,
+    monkeypatch_setenv_from_app_config: Callable,
+) -> Iterable[TestClient]:
+    # test config & env vars ----------------------
+    cfg = deepcopy(app_config)
+    assert cfg["rest"]["version"] == API_VTAG
+    assert cfg["rest"]["enabled"]
+
+    cfg["projects"]["enabled"] = True
+    cfg["director"]["enabled"] = True
+    cfg["exporter"]["enabled"] = True
+
+    monkeypatch_setenv_from_app_config(cfg)
+
+    # app setup ----------------------------------
+    app = create_safe_application(cfg)
+
+    # activates only security+restAPI sub-modules
+    assert setup_settings(app)
+    assert (
+        exporter_settings.get_plugin_settings(app) is not None
+    ), "Should capture defaults"
+
+    setup_db(app)
+    setup_session(app)
+    setup_security(app)
+    setup_rest(app)
+    setup_login(app)
+    setup_users(app)
+    setup_projects(app)
+    setup_exporter(app)  # <---- under test
+    setup_socketio(app)
+    setup_resource_manager(app)
+
+    yield event_loop.run_until_complete(
+        aiohttp_client(
+            app,
+            server_kwargs={"port": cfg["main"]["port"], "host": cfg["main"]["host"]},
+        )
+    )
+
+
+@pytest.fixture
+async def user(client: TestClient) -> AsyncIterable[UserInfoDict]:
+    async with LoggedUser(client=client) as user_info_dict:
+        yield user_info_dict
+
+
+@pytest.fixture
+async def project(
+    client: TestClient, template_path: Path, user: UserInfoDict, product_name: str
+) -> AsyncIterable[ProjectDict]:
+    project = await _new_project(
+        client=client,
+        user=user,
+        product_name=product_name,
+        project_template_file=template_path,
+    )
+    yield project
+    await delete_all_projects(client.app)
+
+
+@pytest.fixture
+def temp_dir(tmpdir: Path) -> Path:
```

Comment:
Use `temp_path` fixture instead of the old `tmpdir`
SEE https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-fixture